package.json

```diff
@@ -11,24 +11,40 @@
 		"dist/color.legacy.cjs.map",
 		"dist/color.legacy.js",
 		"dist/color.legacy.js.map",
-		"src/"
+		"src/",
+		"types/dist/",
+		"types/src/",
+		"types/index.d.ts"
```

Comment:
I just kept this structure when I copied the types over from DefinitelyTyped, but I think doing it this way also makes testing easier since everything's in the same place